tests/unit/test_business_logic.py

```diff
@@ -2138,12 +2143,67 @@ def setup_empty_release_request():
     return release_request, path, author
 
 
-def test_get_visible_comments_for_group(bll):
+@dataclass
+class VisibleItems:
+    """Helper class to make assertions about visiblity of comments and audit logs."""
+
```

Comment:
This class made sense to me after I'd read its use in the tests, but not really before. It might just need some better comments to explain what comments/audits are here - the visible items for a specific group and user at a stage in a request